### PR TITLE
Wiremock

### PR DESCRIPTION
**Pull request by skiddykong**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```
Comment (purple52) [inline pom.xml]: All version numbers should be specified in common bom - this one isn't needed.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```
Comment (purple52) [inline pom.xml]: This dependency shouldn't be here, anyway - should only be in the tests module pom.

File: wiremock-service-test/pom.xml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.0.1</version>
```
Comment (purple52) [inline wiremock-service-test/pom.xml]: Should use version from common bom - check if there is a different jax-rs client you can use already defined in the common bom.

File: wiremock-service-test/pom.xml [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-rs-client</artifactId>
+            <version>3.1.5</version>
```
Comment (purple52) [inline wiremock-service-test/pom.xml]: As above.

File: wiremock-service-test/pom.xml [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>1.0.2.MoJ.Fork</version>
```
Comment (purple52) [inline wiremock-service-test/pom.xml]: Should move this into a property at the top level of the project.

File: wiremock-service-test/pom.xml [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+                        <goals>
+                            <goal>shutdown</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+            </plugin>
```
Comment (purple52) [inline wiremock-service-test/pom.xml]: Don't think you need this here - it is already added in the top level pom.

File: wiremock-service/pom.xml [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        <artifactId>wiremock-service-parent</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>wiremock-service</artifactId>
+    <packaging>war</packaging>
+
+    <build>
+        <finalName>wiremock-service</finalName>
+    </build>
```
Comment (purple52) [inline wiremock-service/pom.xml]: Should use ejb-jar.xml instead.

File: wiremock-service/pom.xml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+    <dependencies>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
```
Comment (purple52) [inline wiremock-service/pom.xml:17]: should be scope provided.

File: wiremock-service/pom.xml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
```
Comment (purple52) [inline wiremock-service/pom.xml]: shouldn't be here

File: wiremock-service-test/pom.xml (hunk 3 above)
Comment (skiddykong) [inline wiremock-service-test/pom.xml]: changed to common dependency

File: wiremock-service-test/pom.xml (hunk 4 above)
Comment (skiddykong) [inline wiremock-service-test/pom.xml]: done

File: wiremock-service-test/pom.xml (hunk 5 above)
Comment (skiddykong) [inline wiremock-service-test/pom.xml]: done

File: wiremock-service/pom.xml (hunk 7 above)
Comment (skiddykong) [inline wiremock-service/pom.xml]: done

File: wiremock-service/pom.xml (hunk 8 above)
Comment (skiddykong) [inline wiremock-service/pom.xml:17]: done

File: wiremock-service/pom.xml (hunk 9 above)
Comment (skiddykong) [inline wiremock-service/pom.xml]: done

**Pull request merged**